topi/python/topi/nn/conv2d.py

```diff
@@ -575,6 +575,11 @@ def group_conv2d_nchw(Input, Filter, stride, padding, dilation, groups, out_dtyp
     else:
         dilation_h, dilation_w = dilation
 
+    if Input.dtype == 'int8':
+        tag = 'group_conv2d_NCHWc_int8'
```

Comment:
This is generic compute function is handle any dtype and should not be tagged as `NCHWc`. There is another implementation optimized for int8 for CUDA backend.

Reply:
Sure, I've just removed that. Thanks for pointing it out.